modules/faq/pages/release-notes.adoc

```diff
@@ -1,5 +1,29 @@
 = Release Notes
 
+== TigerGraph ML Workbench 1.3
+
+=== New features
+
+* Two new xref:1.3@pytigergraph:gds:dataloaders.adoc[data loader classes]: `HGTLoader` and `NodePiece`.
+* Callback functions to all xref:1.3@pytigergraph:gds:gds.adoc[data loader factories]: users can write functions to process the batch in a background thread before it is passed into the training loop.
+* A delimiter parameter to all xref:1.3@pytigergraph:gds.adoc[data loader factories]: users can choose what delimiter they want to separate attributes they are loading from the graph.
+* `customizeHeader()` function to change default timeout and response size values.
```

Comment:
@parkererickson , is this customizeHeader function included in the docs?

Reply:
Oh, no. `customizeHeader()` was part of a patch version I believe. Let me double check though, we can release it as a v1.4 feature too.

Reply:
Sounds good. Let's remove this line then. Thanks!

Reply:
```suggestion
```

Reply:
Will delete customizeHeader() from 1.3 release notes